esm-postbuild.sh

```diff
@@ -0,0 +1,5 @@
+cat >dist/package.json <<!EOF
+{
+    "type": "commonjs"
+}
+!EOF
```

Comment:
oof, it seems unfortunate that an hackish little script like this is necessary to support dual-format packages. there's really no simpler way?

Reply:
I don't know of any other way actually. An alternative is to use different extensions (e.g. `.cjs`), but this would be more complicated to setup. We can also inline the script in the `package.json` if you prefer this.

Reply:
An alternative could be to just commit the `package.json` into the repository (inside a `dist` folder)

Reply:
I pushed another version with the simplified build

Reply:
good idea! for simplicity i think i'll move the additional `package.json` to the project root under a different name (rather than unignoring it and using `find`. but i'll make that change after merging rather than requesting more changes from you!